pkg/asset/installconfig/ovirt/validaton_test.go

```diff
@@ -83,3 +83,36 @@ func Test_validateAuth(t *testing.T) {
 func CreateMockOvirtServer(handler http.HandlerFunc) *httptest.Server {
 	return httptest.NewServer(handler)
 }
+
+func Test_validateURL(t *testing.T) {
+	tests := []struct {
+		url           string
+		expectSuccess bool
+	}{
+		{
+			url:           "engine.example.com",
+			expectSuccess: false,
```

Comment:
test for actual errors and not just error yes/no

Reply:
Changed expectations using the Error returned by URIWithProtocol, hope I understood correctly your point.